src/storage/src/hummock/sstable/mod.rs

```diff
@@ -147,7 +148,7 @@ impl Sstable {
         };
         if enable_bloom_filter() && self.has_bloom_filter() {
             let hash = xxh32::xxh32(dist_key, 0);
-            self.surely_not_have_hashvalue(hash)
+            self.surely_not_have_hashvalue(hash, 0_u32)
```

Comment:
`surely_not_have_dist_key` is only used for test, we can add cfg for `surely_not_have_dist_key`, and add comments for table_id `0`